src/main/java/ca/quickdo/springintro/models/Item.java

```diff
@@ -0,0 +1,54 @@
+package ca.quickdo.springintro.models;
+
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.jackson.Jacksonized;
+
+import javax.persistence.ElementCollection;
+import javax.persistence.Embeddable;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+@Data
+@Jacksonized
```

Comment:
You need to also add a `@Builder` annotation for the `@Jacksonized` annotation to work. You'll keep getting build errors if you don't add the annotation